shared/actions/profile/index.js

```diff
@@ -76,23 +77,29 @@ function* _onUserClick(action: Constants.OnUserClick): SagaGenerator<any, any> {
   if (!username.includes('@')) {
     yield put(switchTo([profileTab]))
     yield put(navigateAppend([{props: {username}, selected: 'profile'}], [profileTab]))
+    return
+  }
+
+  let props = {}
+  const searchResult = yield select(Selectors.searchResultSelector, username)
```

Comment:
I agree it should work like that. This is mostly due to us shoe-horning in support for seeing non-user profiles long after building out the profile page. I think we should ticket it but likely not fix this up in the short term